### ISSUE TITLE
useConfigFile error with 2.1.2

### ISSUE THREAD

**Issue by saknarak** [postcss/postcss-loader#347]
I use nuxt.js

I found that v2.1.2 has problem with useConfigFile option, but v2.1.1 not.

```
ERROR  Failed to compile with 1 errors                                                                 1:55:37 AM

 error  in ./node_modules/bulma/css/bulma.css

Module build failed: ValidationError: PostCSS Loader Invalid Options

options['useConfigFile'] is an invalid additional property


 @ ./node_modules/bulma/css/bulma.css 4:14-118 13:3-17:5 14:22-126
 @ ./.nuxt/App.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?name=client&reload=true&timeout=30000&path=/__webpack_hmr ./.nuxt/client.js
```

**Comment by saknarak** [comment]
I found ```additionalProperties``` has been changed from ```true``` to ```false``` will be cause of this problem.
When I switch back to ```true```, it's work without problem.

**Comment by derevandal** [comment]
#346

**Comment by ai** [comment]
I reverted `additionalProperties` and released 2.1.3
